sensor/models.py

```diff
@@ -29,6 +29,42 @@ class Location(Model):
     def __unicode__(self):
         return self.name
 
+    @staticmethod
+    def _assert_field_for_create_(dict_key, data, err=None):
+        if dict_key not in data:
+            raise KeyError('Needs %s to create location!' % (err if err else dict_key))
+
+    @classmethod
+    def create(cls, data):
+        _assert_field_for_create('id', data, err='device id')
+        _assert_field_for_create('key', data, err='api key')
+        _assert_field_for_create('latitude', data)
+        _assert_field_for_create('longitude', data)
+        _assert_field_for_create('altitude', data)
```

Comment:
The `altitude` field is currently not required in the `Location` model. If we make it required in this input validation I think we should start changing the `Location`model to make the `altitude` field required.